.github/workflows/compile.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-go@v5
         with:
-          go-version: stable
+          go-version-file: 'go.mod'
```

Comment:
We can drop the quotes, since this is yaml file and this value is a yaml string.

Reply:
I changed it.